website/docs/r/virtual_machine.html.markdown

```diff
@@ -342,10 +129,9 @@ The following arguments are supported:
 * `boot_diagnostics` - (Optional) A boot diagnostics profile block as referenced below.
 * `vm_size` - (Required) Specifies the [size of the virtual machine](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-size-specs/).
 * `storage_image_reference` - (Optional) A Storage Image Reference block as documented below.
-* `storage_os_disk` - (Required) A Storage OS Disk block as referenced below.
-* `delete_os_disk_on_termination` - (Optional) Flag to enable deletion of the OS disk VHD blob or managed disk when the VM is deleted, defaults to `false`
+* `storage_os_disk` - (Required) A Storage OS DFlag to enable deletion of the OS disk VHD blob when the VM is deleted, defaults to `false` (not yet supported).
 * `storage_data_disk` - (Optional) A list of Storage Data disk blocks as referenced below.
-* `delete_data_disks_on_termination` - (Optional) Flag to enable deletion of storage data disk VHD blobs or managed disks when the VM is deleted, defaults to `false`
+* `delete_data_disks_on_termination` - (Optional) Flag to enable deletion of storage data disk VHD blobs or managed disks when the VM is deleted, defaults to `false` (not yet supported).
```

Comment:
we should just outright remove reference to managed disks if they're not supported imo, the phrasing here isn't clear what's not supported

Reply:
Should I remove all the attributes related to managed disks? Until support for them comes to AzureStack?

Reply:
I think that’d be good as the documentation should only supposed to show things that are supported / available in the provider (rather than things which aren’t)

Reply:
(As you say - we can add them back in when support is available in azure stack)